lib/trento/users/user.ex

```diff
@@ -53,6 +53,14 @@ defmodule Trento.Users.User do
     |> lock_changeset(attrs)
   end
 
+  def profile_update_changeset(user, attrs) do
+    user
```

Comment:
Shouldn't we have `pow_current_password_changeset` here?
How do we enforce otherwise that the `current_password` is coming?
Or `pow_password_changeset` does this?  But in that the case, it would happen in the current update, which doesn't

Reply:
you're right, done!